packages/infra/lib/api-stack/fhir-converter-connector.ts

```diff
@@ -32,7 +32,7 @@ function settings() {
     // Number of messages the lambda pull from SQS at once
     lambdaBatchSize: 1,
     // Max number of concurrent instances of the lambda that an Amazon SQS event source can invoke [2 - 1000].
-    maxConcurrency: fhirConverterCPUAmount * fhirConverterTaskCounMin * 4,
+    maxConcurrency: fhirConverterCPUAmount * fhirConverterTaskCounMin,
```

Comment:
nit: `fhirConverterTaskCounMin` missing T in Count. Not a blocker

Reply:
Will fix on a follow-up PR